### ISSUE TITLE
When the pipelines service is restarted, the list of models in `open-webui` disappears.

### ISSUE THREAD

**Issue by sabretus** [open-webui/pipelines#111]
# Bug Report

## Description

**Bug Summary:**
When the `pipelines` service is restarted, the list of models in `open-webui` disappears.

**Steps to Reproduce:**
Everything works fine until the `pipelines` container is restarted. Then all models are gone from `open-webui`. 

**Expected Behavior:**
Automatically reload the list of models

**Actual Behavior:**
I have to go to settings -> pipelines -> hit SAVE to load list of models

## Environment

- **Open WebUI Version:** 0.3.5
- **Open WebUI Pipelines Version:** latest from master
- **Ollama (if applicable):** none

- **Operating System:** Ubuntu 22.04.4

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots
No errors in logs

## Installation Method
Service open-webui configured with pipelines (with litellm_manifold_pipeline.py) and connected to lightllm proxy. All are running in docker containers

**Comment by bannert1337** [comment]
Do you have a volume mapping for pipelines?

**Comment by attilaszasz** [comment]
@bannert1337, I experience the same problem. I have a volume mapping, but it's not the same as the open-webui volume. Should it be?

My volumes in docker-compose:

```
volumes:
  open-webui: {}
  pipelines: {}
```

**Comment by justinh-rahb** [comment]
This is likely an issue from the pipeline code itself not implementing the current initialization/startup logic.

**Comment by attilaszasz** [comment]
Yep, I fixed for the Google pipeline, the same fix should work here, but I don't use litellm to be able to test it.

**Comment by justinh-rahb** [comment]
@attilaszasz thanks again for that, I've applied the same in #210 to fix LiteLLM manifold. Would appreciate if someone here would test that before merging.
